apps/prometheus/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
We store only applications that are based on NSM SDK in the folder apps/. Please move this near the example.